contracts/core/modules/RebalanceAuctionModule.sol

```diff
@@ -76,11 +76,24 @@ contract RebalanceAuctionModule is
      */
     function bid(
         address _rebalancingSetToken,
-        uint256 _quantity
+        uint256 _quantity,
+        bool _executePartialQuantity
```

Comment:
I think you had the naming right in the comment below. Maybe `allowPartialFill` or `shouldPartiallyFill`?